src/data/table/database.py

```diff
@@ -9,8 +9,12 @@ def pg_timestamp(timestamp: datetime):
 
 
 class DatabaseOperation(Enum):
-    ADDED = 0
-    EDITED = 1
+    # No operation was performed (the record didn't need updating)
```

Comment:
In Python, you can write document strings for any element.
You can do this  by typing """my documentation""" under the element.
This also adds the documentation in the IDE if it's supported

Reply:
Example: 
`class DatabaseOperation(Enum):`
`    NONE = 0,  'No operation was performed (the record didn't need updating)'`
or
`class DatabaseOperation(Enum):`
`    NONE = 0`
`    """No operation was performed (the record didn't need updating)"""`